src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -47,8 +47,7 @@ public ResponseEntity<Object> createPartnership(
         var transactionId = transaction.getId();
         var logMap = new HashMap<String, Object>();
         logMap.put(URL_PARAM_TRANSACTION_ID, transactionId);
-
-        try {
+    try {
```

Comment:
Incorrect indentation. Revert the file as this is the only change.

Reply:
file reverted